Stripe/PublicHeaders/STPCustomer.h

```diff
@@ -50,6 +52,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, readonly) NSArray<id<STPSourceProtocol>> *sources;
 
+/**
+ *  The customer's shipping address.
+ */
+@property(nonatomic, readonly) STPAddress *shipping;
```

Comment:
👍 – I called it shipping to match the API object, but I think you're right that it's better to use shippingAddress